simpa/utils/libraries/structure_library/StructureBase.py

```diff
@@ -59,7 +59,9 @@ def __init__(self, global_settings: Settings,
         if Tags.PRIORITY in single_structure_settings:
             self.priority = single_structure_settings[Tags.PRIORITY]
 
-        self.partial_volume = single_structure_settings[Tags.CONSIDER_PARTIAL_VOLUME]
+        if Tags.CONSIDER_PARTIAL_VOLUME in single_structure_settings:
+            self.partial_volume = single_structure_settings[Tags.CONSIDER_PARTIAL_VOLUME]
+        else: self.partial_volume = False
```

Comment:
I think our coding convention is to have if else block statements on a new line unless it is an inline if-else. Apart from that the changes look reasonable.

Reply:
Alright, I changed this.